.github/workflows/unittests.yml

```diff
@@ -137,6 +137,48 @@ jobs:
     - name: Run tests
       run: tox -e cryptography
 
+  utscapy-scanner:
```

Comment:
I believe it should be able to integrate this test with the other one, using something like
```
continue-on-error: ${{matrix...}
.......
UTscapy: ...
```